pkg/crc/machine/machine.go

```diff
@@ -537,6 +537,18 @@ func Status(statusConfig ClusterStatusConfig) (ClusterStatusResult, error) {
 	}
 
 	if IsRunning(vmStatus) {
+		_, crcBundleMetadata, err := getBundleMetadataFromDriver(host.Driver)
+		if err != nil {
+			result.Error = err.Error()
+			return *result, errors.Newf("Error loading bundle metadata: %v", err)
+		}
+		proxyConfig, err := getProxyConfig(crcBundleMetadata.ClusterInfo.BaseDomain)
+		if err != nil {
+			result.Error = err.Error()
+			return *result, errors.Newf("Error getting proxy configuration: %v", err)
+		}
+		proxyConfig.ApplyToEnvironment()
```

Comment:
same here shouldn't we first check if proxy is enable before apply to env?

Reply:
I can add it if you want, `proxyConfig.ApplyToEnvironment()` starts with `if !proxy.IsEnabled() return`

Reply:
@cfergeau Ah right :( no need